File: services/forms/app/schemas/ben2.py
from datetime import datetime
from typing import Optional

from pydantic import BaseModel

class Ben2Base(BaseModel):
    ref_user: Optional[int] = None
    company_id: Optional[int] = None
    cin: Optional[str] = None
    company_name: Optional[str] = None
    company_address: Optional[str] = None
    company_email: Optional[str] = None
    reporting_company: Optional[bool] = None
    declaration_ownership: Optional[bool] = None
    change_ownership: Optional[bool] = None
    change_exiting: Optional[bool] = None
    change_reporting_company: Optional[bool] = None
    cin_holding_company: Optional[int] = None
    name_holding_company: Optional[str] = None
    number_of_significant_owners: Optional[int] = None
    copy_attachment: Optional[str] = None
    optional_attachment: Optional[str] = None
    resolution_number: Optional[int] = None
    resolution_date: Optional[datetime] = None
    designation: Optional[str] = None
    signitory: Optional[str] = None
    din_pan: Optional[str] = None
    category: Optional[str] = None
    name_of_pcs: Optional[str] = None
    associate_or_fellow: Optional[str] = None
    membership_number: Optional[int] = None
    ref_user_name: Optional[str] = None

class Ben2Create(Ben2Base):
    pass

class Ben2Read(Ben2Base):
    id: int
    created_at: datetime

    class Config:
        from_attributes = True

File: legacy/ComplyCrafter_Data/Forms/BEN2.cs
using AlphaUtil.Core.Models;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace ComplyCrafter_Data
{
    [TableName("tbl_ben2", "vw_ben2")]
    public class BEN2 : BaseModel
    {
        [Column("ref_user")]
        public int? RefUser { get; set; }

        [Column("company_id")]
        public int? CompanyId { get; set; }

        [Column("cin")]
        public string? Cin { get; set; }

        [Column("company_name")]
        public string? CompanyName { get; set; }

        [Column("company_address")]
        public string? CompanyAddress { get; set; }

        [Column("company_email")]
        public string? CompanyEmail { get; set; }
        [Column("reporting_company")]
        public Boolean? ReportingCompany { get; set; }

        [Column("declaration_ownership")]
        public Boolean? DeclarationOwnership { get; set; }

        [Column("change_ownership")]
        public Boolean? ChangeOwnership { get; set; }

        [Column("change_exiting")]
        public Boolean? ChangeExiting { get; set; }

        [Column("change_reporting_company")]
        public Boolean? ChangeReportingCompany { get; set; }

        [Column("cin_holding_company")]
        public int? CinHoldingCompany { get; set; }

        [Column("name_holding_company")]
        public string? NameHoldingCompany { get; set; }

        [Column("number_of_significant_owners")]
        public int? NumberOfSignificantOwners { get; set; }

        [Column("copy_attachment")]
        public string? CopyAttachment { get; set; }

        [Column("optional_attachment")]
        public string? OptionalAttachment { get; set; }

        [Column("resolution_number")]
        public int? ResolutionNumber { get; set; }

        [Column("resolution_date")]
        public DateTime? ResolutionDate { get; set; }

        [Column("designation")]
        public string? Designation { get; set; }

        [Column("signitory")]
        public string? Signitory { get; set; }

        [Column("din_pan")]
        public string? DinPan { get; set; }

        [Column("category")]
        public string? Category { get; set; }

        [Column("name_of_pcs")]
        public string? NameOfPcs { get; set; }

        [Column("associate_or_fellow")]
        public string? AssociateOrFellow { get; set; }

        [Column("membership_number")]
        public int? MembershipNumber { get; set; }

    }

    [TableName("tbl_ben2", "vw_ben2")]
    public class BEN2View : BEN2
    {
        [Column("ref_user_name")]
        public string? RefUserName { get; set; }
    }

}